static/app/gettingStartedDocs/android/android.tsx

```diff
@@ -64,6 +64,9 @@ const getConfigurationSnippet = (params: Params) => `
   <!-- Required: set your sentry.io project identifier (DSN) -->
   <meta-data android:name="io.sentry.dsn" android:value="${params.dsn.public}" />
 
+  <!-- Add data like request headers and device name, see https://docs.sentry.io/platforms/android/data-management/data-collected/ for more info -->
+  <meta-data android:name="io.sentry.send-default-pii" android:value="true" />
```

Comment:
There's also the wizard where I couldn't find any trace of the PII setting being suggested / enabled.

Reply:
I opened an issue for the wizard changes: https://github.com/getsentry/sentry-wizard/issues/790